apps/beets-frontend-v3/package.json

```diff
@@ -30,7 +30,7 @@
     "@nikolovlazar/chakra-ui-prose": "^1.2.1",
     "@repo/lib": "workspace:*",
     "@sentry/nextjs": "8.50.0",
-    "date-fns": "^2.30.0",
+    "date-fns": "^4.1.0",
```

Comment:
This seems like a huge jump, have we checked that no new changes are messing with the app?

Reply:
that's why `date-fns` recommended running a type check after the upgrade (which caught one error)